chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+echo "Preparing for uninstalling apps left by Rancher in the following namespaces:" "$namespaces"
+for namespace in $namespaces; do
+  # delete the helm operator pods
+  pod_list=$(kubectl get pods -n "$namespace" -o name --ignore-not-found)
+  for pod in $pod_list; do
+    if [[ $pod =~ ^pod\/helm-operation-* ]]; then
+      echo "--- deleting the pod [$pod]"
+      kubectl delete "$pod" -n "$namespace"
+    fi
+  done
+  app_list=$(helm list -n "$namespace" -q)
+  for app in $app_list; do
+    if [[ $app =~ .crd$ ]]; then
+      echo "--- skip" $app
+      continue
+    fi
+    echo "--- deleting the app [$app] in the namespace [$namespace]"
+    helm uninstall "$app" -n "$namespace"
+    t=0
+    # by default, wait 120 seconds in total for an app to be uninstalled
+    while [[ "$(kubectl get pods --selector app="$app" -n "$namespace" -o json | jq '.items | length')" != "0" ]] && [[ $t -lt "$timeout" ]]; do
+      echo "waiting for pods of [$app] to be terminated ..."
+      sleep 5
+      t=$((t + 5))
+    done
+    if [[ "$(kubectl get pods --selector app="$app" -n "$namespace" -o json | jq '.items | length')" != "0" ]]; then
+      echo "timeout uninstalling [$app] in the namespace [$namespace]"
+    else
+      echo "uninstalled [$app] in the namespace [$namespace]"
+    fi
+  done
+done
+
+echo "The job is done."
```

Comment:
This should list failed jobs if any, or:
```suggestion
echo "Cleanup finished successfully."
```

Reply:
fixed.